src/components/Modal.tsx

```diff
@@ -0,0 +1,87 @@
+import "./Modal.scss";
+
+type ModalProps = {
+  selector: string;
+  size: "sm" | "md" | "lg";
+  prevent?: boolean;
+};
+
+const Modal = ({ selector, size, prevent }: ModalProps) => {
+  return (
+    <tds-modal
+      ref={(element) => {
+        /* if (size === "lg") {
```

Comment:
Do we still want to keep this commented out code?

Reply:
Yes I would like to keep it until we have fixed the issue :)